zod/package.json

```
@@ -11,7 +11,7 @@
   "types": "dist/index.d.ts",
   "license": "MIT",
   "peerDependencies": {
-    "react-hook-form": "^7.0.0",
+    "react-hook-form": "7.55.0",
```

Should this be locked to version 7.55.0?
```suggestion
    "react-hook-form": "^7.55.0",
```

---

Good catch @spiftire